### PR TITLE
feat(node-watcher): Store motions in chain-db

### LINKED ISSUE #148: [Node watcher] Track Council motions

### PR DESCRIPTION
**Pull request by Tbaut**
- closes #148

### CODE REVIEW

File: back/node_watcher/src/tasks/createMotion.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import { prisma } from '../generated/prisma-client';
+import { motionStatus } from '../util/statuses';
+import { NomidotMotion, NomidotMotionRawEvent, Task } from './types';
+
+const l = logger('Task: Motions');
+
+/*
+ *  ======= Table (Motion) ======
+ */
+const createMotion: Task<NomidotMotion[]> = {
```
Comment (yjkimjunior) [inline back/node_watcher/src/tasks/createMotion.ts]: ah wait btw, these are PolkassemblyMotion! can go do a replacement in another PR to rename all the Polkassembly stuff as such though.

Comment (Tbaut) [reply back/node_watcher/src/tasks/createMotion.ts]: Sure. tbh I'd rather name that all "NomidotBlabla" because we do share things already such as blockbnumbrer, and the future might add some more..

Comment (yjkimjunior) [reply back/node_watcher/src/tasks/createMotion.ts]: in that case then we should remove Nomidot prefix entirely imo

Comment (Tbaut) [reply back/node_watcher/src/tasks/createMotion.ts]: Agree, let's use another PR for that though :smile:
